addons/web/static/src/views/fields/text/text_field.js

```diff
@@ -131,7 +140,7 @@ export class ListTextField extends TextField {
         return 0;
     }
     get rowCount() {
-        return 1;
+        return this.props.rowCount || 1;
```

Comment:
Shouldn't we rather change the default value in the props? As the default value is 2 for TextField, this will always override the 1 here, no?

Reply:
Oh indeed. 😅😬